shared_python/FinalTables.py

```diff
@@ -5,37 +5,44 @@
 
 
 class FinalTables(object):
-
     def __init__(self, args, sql: Sql, log: Logger):
         self.args = args
         self.sql = sql
         self.original_database = args.temp_db_database
         self.final_database = args.output_database
         self.log = log
 
-    def original_table(self, table_name, filter='', database_name=None):
+    def original_table(self, table_name, filter="", database_name=None):
         if table_name is None:
             return None
         if database_name is None:
             original_database = self.original_database
         else:
             original_database = database_name
-        query = "SELECT * FROM `{0}`.`{1}` {2}".format(original_database, table_name, filter)
+        query = "SELECT * FROM `{0}`.`{1}` {2}".format(
+            original_database, table_name, filter
+        )
         return self.sql.execute_dict(query)
 
     def _escape_unescape(self, item):
-        return html.unescape(item).replace('\\', '\\\\').replace('"', '\\"').replace("'", "\\'").replace("%", "%%")
+        return (
+            html.unescape(item)
+            .replace("\\", "\\\\")
+            .replace('"', '\\"')
+            .replace("'", "\\'")
+            .replace("%", "%%")
+        )
 
     def _value(self, row):
         value = []
         for item in row:
-            if type(item) is str:
+            if type(item) is str:  # noqa: E721
```

Comment:
Yup!  That's actually the right fix; I just didn't want to mix whitespace only changes with substantive ones in this patch, since it's hard enough to understand already.  I'll make another PR to fix this one and the other isinstance underneath.

Reply:
Makes sense 👍